icons/session.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Session</title><path d="m19.4 12.2-4.5-2.5H18c2.7 0 4.8-2.2 4.8-4.8S20.6 0 18 0H7.3C3.9 0 1.2 2.7 1.2 6.1c0 2.4 1.3 4.6 3.4 5.7l4.5 2.5H6c-2.7 0-4.8 2.2-4.8 4.8S3.4 24 6 24h10.7c3.3 0 6.1-2.7 6.1-6.1 0-2.4-1.3-4.6-3.4-5.7zM5.3 10.5C3.7 9.6 2.7 8 2.7 6.2c-.1-2.6 2.1-4.7 4.7-4.7h10.5c1.8 0 3.3 1.4 3.4 3.2.1 1.9-1.4 3.5-3.3 3.5h-6.1c-.4 0-.7.3-.7.7v4.9l-5.9-3.3zm11.3 12H6.1a3.4 3.4 0 0 1-3.4-3.2c-.1-1.9 1.4-3.5 3.3-3.5h6c.4 0 .7-.3.7-.7v-4.9l6 3.3c1.6.9 2.6 2.5 2.6 4.3.1 2.6-2.1 4.7-4.7 4.7z"/></svg>
```

Comment:
You may need to use a higher precision while using SVGO. Right now, there are significant differences. Ex. 
![image](https://github.com/simple-icons/simple-icons/assets/12817388/ed89f458-e3d3-4f6a-b0a7-21424fc9e0c6)

Can you take another look at it?